src/components/podcasts/card.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {FunctionComponent} from 'react'
+import {PodcastResource} from 'types'
+import Image from 'next/image'
+
+type PodcastCardProps = {
+  title: string
+  path: string
+  imageUrl: string
+  contributors: Array<string>
+}
+
+const PodcastCard: FunctionComponent<PodcastCardProps> = ({
+  title,
+  path,
+  imageUrl,
+  contributors,
+}) => (
+  <li className="list-none bg-white p-3 text-center max-w-xs shadow-md rounded-md transform transition-transform duration-300 hover:scale-105">
+    <a title="View podcast" href={path} className="flex flex-col h-full">
+      <div className="mb-0 h-cardimage w-cardimage flex-grow-0">
```

Comment:
instead of globally applying a size in the tailwind config, we might consider using emotion for this singular case of a specified size. It would look like:

```
css={{
  width: '200px',
  height: '200px'
}}
```

I believe you need:

```
import {jsx} from '@emotion/core'```

up top as well for the `css` prop to function, but that's its intent and why emotion is in the codebase.

We generally want to keep tailwind config very non-specific and reserved for truly global concerns

Reply:
Yeah, I wasn't sure how to do this in an elegant way. Either, using a predefined size from tailwind like `h-64` && `w-64` and then setting the `width` and `height` as `64 x 16` on the `Image` component.

Is there a nice way in the repo to create a shared value? 

Essentially, a `size` that is an Integer that can also be used as a `px` value by this `div`?

Reply:
Ahh right, I see 👍